doc/sphinx/addendum/nsatz.rst

```diff
@@ -73,6 +73,57 @@ files that declare the axioms used to define the real numbers, you can
    See the file `Nsatz.v <https://github.com/rocq-prover/stdlib/blob/master/test-suite/success/Nsatz.v>`_
    for examples, especially in geometry.
 
+.. tacn:: ensatz {? with strategy := @one_term }
+
+   This tactic is for solving goals of the form
+
+   :math:`\begin{array}{l}
+   \forall X_1, \ldots, X_n \in A, \\
+   P_1(X_1, \ldots, X_n) = Q_1(X_1, \ldots, X_n), \ldots, P_s(X_1, \ldots, X_n) = Q_s(X_1, \ldots, X_n) \\
+   \vdash \exists Y_1, \ldots, Y_m \in A, \\
+   P(X_1, \ldots, X_n) = Y_1 * Q_1'(X_1, \ldots, X_n) + \ldots + Y_m * Q_m'(X_1, \ldots, X_n)
+   \end{array}`
+
+   where :math:`P, P_1, Q_1, Q_1', , \ldots, P_s, Q_s, Q_m'` are polynomials and :math:`A` is an integral
```

Comment:
Would you briefly state the relationship between the `Qs` and the `Q'm`?  If they're not related, maybe pick a variable other than `Q'`.?